### PR TITLE
Log handling

### PR DESCRIPTION
**Pull request by sepidetari**
**Sending logs produced by microservices to Elasticsearch using Logstash and visualize them by Kibana:**

image

__ To run it follow the steps mentioned in the wiki
 [https://github.com/dice-group/sask/wiki/Logging](https://github.com/dice-group/sask/wiki/Logging)

### CODE REVIEW

**Comment by sepidetari** [comment]
@KHaack It works now :)

File: chatbot/src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	<appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<charset>UTF-8</charset>
+			<Pattern>%d{yyyy-MM-dd HH:mm:ss.SSS} %-5level [%thread] "chatbot_ms" [%C] %logger{36} - %msg%n</Pattern>
+		</encoder>
+	</appender>
+
+	<appender name="FILE"
+		class="ch.qos.logback.core.rolling.RollingFileAppender">
+		<file>C:/logs/chatbot.log</file>
```
Comment (prasanthhs) [inline chatbot/src/main/resources/logback-spring.xml]: Windows path used in two places in configuration of chatbot log file. Is this correct?

Comment (sepidetari) [reply chatbot/src/main/resources/logback-spring.xml]: To run ELK in docker a shared volume between them is needed that it is a windows path for now.

**Comment by KHaack** [comment]
we are currently checking the full setup on the staging

**Pull request merged**